SPECS/flatbuffers/flatbuffers.spec

```diff
@@ -6,7 +6,7 @@ parsing/unpacking it first, while still having great forwards/backwards
 compatibility.}
 Summary:        Memory efficient serialization library
 Name:           flatbuffers
-Version:        2.0.8
+Version:        23.5.26
```

Comment:
Maybe we need a rebase to fix Distribution as `Mariner` -> `Azure Linux`

Reply:
Fixed